SmartDeviceLink/SDLSecondaryTransportManager.m

```diff
@@ -705,6 +709,10 @@ - (SDLSecondaryTransportType)sdl_getTransportTypeFromProtocol:(SDLProtocol *)pro
     }
 }
 
+- (void)securityManagerWasSet {
+    self.secondaryProtocol.securityManager = self.primaryProtocol.securityManager;
```

Comment:
Did you test this? It doesn't seem like it'd be enough to restart the secondary transport manager in checking and setting the security manager.

You need to test after updating:

1. Does this work if you don't set a security manager.
2. Does this work if the `TransportEventUpdate` is set before the RAIR is returned.
3. Does this work if the `TransportEventUpdate` is set after the RAIR is returned.

Reply:
this should be resolved now